.github/workflows/integration-tests.yml

```diff
@@ -622,7 +622,7 @@ jobs:
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
         with:
           repository: smartcontractkit/chainlink-solana
-          ref: develop
+          ref: BCF-2612-ChainReader
```

Comment:
- [ ] undo before merge

Reply:
Isn't this required to build core Solana integration tests with from BCF-2612-ChainReader branch?

Although I don't think this is needed here as Solana won't be using chain reader for the integration tests.

Reply:
> Isn't this required to build core Solana integration tests with from BCF-2612-ChainReader branch?

I have never had to do it. I just update the go.mod files.